lib/tom_queue.rb

```diff
@@ -48,5 +48,9 @@ module TomQueue
   class << self
     attr_accessor :exception_reporter
     attr_accessor :logger
+
+    def in_worker?
```

Comment:
Might be worth some commentary as to what this is for (i.e. code can detect if it's in a worker).

I'd also be tempted to mention that the intention this is a placeholder, it would be good to replace with a `current_worker` accessor and this is a boolean check if that is nil.